docs/hardware/devices/seeed-studio/esp32-sx1262-kit/index.mdx

```diff
@@ -5,6 +5,11 @@ sidebar_label: XIAO ESP32S3 & Wio-SX1262 Kit
 sidebar_position: 2
 ---
 
+:::warning[Note]
+The Wio-SX1262 module shipped with this kit does not have the same pin configuration as the one shipped with the XIAO nRF52840 kit.
```

Comment:
@rcarteraz @pdxlocations thoughts on using an admonition for this? I could go either way.